core/http.md

```diff
@@ -0,0 +1,3293 @@
+# xRegistry HTTP Binding
+
+## Abstract
+
+This specification defines an HTTP protocol binding for the
+[xRegistry specification](./spec.md). This document will include just the
+HTTP specific details and semantics, leaving the
+[core specification](./spec.md) to define the xRegistry generic processing
+model and semantics that apply to all protocols.
+
+## Table of Contents
+
+- [Notations and Terminology](#notations-and-terminology)
+  - [Terminology](#terminology)
+- [HTTP Binding Overview](#http-binding-overview)
+- [Registry HTTP APIs](#registry-http-apis)
+  - [Entity Processing Rules](#entity-processing-rules)
+    - [Creating or Updating Entities](#creating-or-updating-entities)
+  - [Registry Entity](#registry-entity)
+    - [`GET /`](#get-)
+    - [`PATCH /`](#patch-)
+    - [`PUT /`](#put-)
+    - [`POST /`](#post-)
+    - [`GET /export`](#get-export)
+  - [Registry Capability](#registry-capability)
```

Comment:
Possibly rename to Registry Capabilities, given that all requests operate on the collection?

Reply:
excellent catch!